tlsfuzzer/messages.py

```diff
@@ -602,7 +608,8 @@ def generate(self, status):
             if cert_req is not None:
                 self.msg_alg = next((sig for sig in
                                      cert_req.supported_signature_algs
-                                     if sig[1] == SignatureAlgorithm.rsa),
+                                     if sig[1] == SignatureAlgorithm.rsa or
```

Comment:
The test case expects a specific list of algorithms enabled: https://github.com/tomato42/tlsfuzzer/pull/124/files#diff-d34a9d1a6a7804d2f614ffd55143b39dR72 You need to pass in the expected values as the `-s` parameter when you run `test-rsa-pss-sigs-on-certificate-verify.py`

The purpose is that this makes sure the expected algorithms are advertised by server, nothing more nothing less. Also allows for catching unexpected changes to that list in the future.